README.md

```diff
@@ -15,6 +15,33 @@ You can publish the views with:
 php artisan vendor:publish --tag=rapidez-sitemap-views
 ```
 
+## Sitemap Generation
+This package includes a configuration option to control the automatic scheduling of the sitemap generation command.
+
+In the package’s service provider, if `schedule_generate_command` is enabled, the command is scheduled:
+
+```php
+if (config('rapidez.sitemap.schedule_generate_command')) {
+    Schedule::command('rapidez:sitemap:generate')->twiceDaily(0, 12);
+}
+```
+
+To disable scheduling, set this in your .env: `RAPIDEZ_SITEMAP_SCHEDULE_GENERATE=false`.
+
+To run the sitemap generation manually, use:
+```bash
+php artisan rapidez:sitemap:generate
+```
```

Comment:
I'm not really sure if we want this within the package. From `rapidez/core` we're not registering any schedules, for example for the indexer: https://docs.rapidez.io/3.x/indexer.html#scheduler and that's moved with Laravel 11 to `routes/console.php` or the `bootstrap/app.php`, maybe we should provide some examples within `rapidez/rapidez` which you can simply uncomment and change?

I'm afraid if we're going to register schedules from packages users aren't aware of them, there is a `schedule:list` command but still; better to have them defined within your project in 1 place? And mention that in the configuration or deployment docs.

Thoughts @kevinmeijer97 / @indykoning / @BobWez98?

Reply:
As we're not doing it for other commands either, and you will always lack control by registering it automatically.
(Like, when do you want it executed? Do you want to run it in your own cronjob so you can separate command output?)

I agree it might be better to make users register it manually so they have maximum control.
That way they could even execute it on events and not based on cron at all.

Reply:
Agreed, i removed the scheduling and added some info in the readme for this